util/a2e/atoe.c

```diff
@@ -158,8 +158,10 @@ char e2a_tab[CONV_TABLE_SIZE];
  */
 #include "atoe.h"
 
-int   atoe_vsnprintf(char *str, size_t count, const char *fmt, va_list args);
 int   atoe_fprintf(FILE *, const char *, ...);
+int   atoe_vfprintf(FILE *, const char *, va_list);
+int   atoe_vsnprintf(char *str, size_t count, const char *fmt, va_list args);
+int   atoe_vsprintf(char *, const char *, va_list);
```

Comment:
Suggest the parameters should have names or be improved:
```c
int   atoe_fprintf(FILE *file, const char *fmt, ...);
int   atoe_vfprintf(FILE *file, const char *fmt, va_list args);
int   atoe_vsnprintf(char *buf, size_t buflen, const char *fmt, va_list args);
int   atoe_vsprintf(char *buf, const char *fmt, va_list args);
```

Reply:
Updated to reflect suggested changes.